.github/workflows/documentation.yml

```diff
@@ -0,0 +1,78 @@
+name: Update Documentation
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  documentation:
+    # Generate documentation and push to gh-pages branch using the predefined JDK versions in the strategy section
+    name: Documentation
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [ 8.x ]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+
+      - run: npm install
+      - run: npm run docs
+
+      - name: Copy to documentation Website Location
+        run: |
+          mkdir ../gh-pages
+          cp -r docs/* ../gh-pages/
+          cd ../gh-pages
+
+      - name: Commit documentation changes
+        run: |
+          if [[ `git status --porcelain` ]]; then
+            echo "Commit documentation"
+            # Set identity
+            git config --global user.email "dev-git-action@paypal.com"
+            git config --global user.name "Git Action Doc"
+          
+            # Add branch
+            git checkout -B gh-pages
+          
+            # Push generated files
+            git add -A
+            git commit -m "Documentation updated"
+            git push origin gh-pages -fq > /dev/null
+          fi
+  code-coverage:
```

Comment:
Question: How come this is added here too in the documentation task?  Isn't coveralls already running on master branch in the above workflow (NodeJS SDK CI)?

Reply:
Good catch. Let me remove this one. 
I added originally in here until I realise that it's good to run test coverage over all branches.